spec/features/admin/admin_can_see_users_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe 'Admin' do
+
+  it 'restricts paths to non admin' do
+      visit '/admin/dashboard'
+      expect(page.status_code).to eq(404)
+  end
+
+  it 'shows list of users on dashboard' do
+    admin = User.create(first_name: 'Woodrow', last_name: 'Wilson', email: 'ww1@gmail.com', role: 1, token: 'dasf', refresh_token: 'dasf')
+    user = User.create(first_name: 'Jimmy', last_name: 'Carter', email: 'peanut_farmer@gmail.com', role: 0)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+    visit '/admin/dashboard'
+
```

Comment:
Should this test be stubbing out Omniauth sign in instead of ApplicationController? Something like this (as we did for the paired project?: 
OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new(auth_hash)